code/modules/reagents/reagent_containers/food/snacks/cultural/human.dm

```diff
@@ -417,3 +462,115 @@
 	reagent_data = list(/singleton/reagent/nutriment = list("blue raspberry" = 5, "white chocolate" = 3))
 	drop_sound = 'sound/items/drop/glass.ogg'
 	pickup_sound = 'sound/items/pickup/glass.ogg'
+
+
+// Eridani
+
+/obj/item/reagent_containers/food/snacks/bowl
```

Comment:
Mark this as abstract if it is not supposed to be spawned

`abstract_type = /obj/item/reagent_containers/food/snacks/bowl`

Reply:
do i just write abstract_type in front of it or do i write it in the details of the object?

Reply:
It's a variable, you set it inside the object itself

Reply:
marked, thanks!